### PR TITLE
[Owners] Update Documentation Links in Examples

### PR DESCRIPTION
**Pull request by kreimnitz**
### What does this Pull Request accomplish?

Fix up the example links to match the update docs links.  Add link to online help.

### Why should this Pull Request be merged?

Fixes broken links

### What testing has been done?

Tested new links

### CODE REVIEW

File: examples/niscope/fetch-continuous.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 # The gRPC API is built from the C API.  NI-SCOPE documentation is installed with the driver at:
 # C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niScope\Documentation\English\Digitizers.chm
 #
+# A version of this .chm is available online at:
+# Link: https://zone.ni.com/reference/en-XX/help/370592AB-01/
```
Comment (kylesull30) [inline examples/niscope/fetch-continuous.py:8]: I guess we stopped posting help online after 16.1?

File: examples/niswitch/controlling-an-individual-relay.py [hunk 2]
```diff
@@ -1,16 +1,20 @@
-# Example communication with NI-SWITCH over gRPC
+# This example demonstrates how to control an individual relay on a module.
 #
 # The gRPC API is built from the C API.  NI-SWITCH documentation is installed with the driver at:
 # C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niSwitch\Documentation\English\SWITCH.chm
 #
+# A version of this .chm is available online at:
+# Link: https://zone.ni.com/reference/en-XX/help/375472H-01/
```
Comment (kylesull30) [inline examples/niswitch/controlling-an-individual-relay.py:7]: This one looks a little bit older. I'm not saying I know if there is a new version or not, just wanted to make sure it is intentional.

**Review by kylesull30 — APPROVED** [review]

File: examples/niscope/fetch-continuous.py (hunk 1 above)
Comment (kreimnitz) [reply examples/niscope/fetch-continuous.py:8]: Yeah, this is the latest available version for now.

File: examples/niswitch/controlling-an-individual-relay.py (hunk 2 above)
Comment (kreimnitz) [reply examples/niswitch/controlling-an-individual-relay.py:7]: I think this is the latest.  These were the links sent to me by Curtis.  I'll follow up with him to ensure they're the latest

**Review by jonlugoatx — APPROVED** [review]

**Pull request merged**